app/views/pages/_lesson_card.html.erb

```diff
@@ -29,8 +29,18 @@
   <div class="text-center ">
     <div class="mt-2 "<span style="color: gray;font-size:10px; font-style:italic;">Created by <%= lesson.searchable.course.user.name %></span></div>
     <div class="mt-2 mb-4 "<span style="color: gray;font-size:10px; font-style:italic;">Last Edited on <%= lesson.updated_at.to_date.strftime("%B %d, %Y")%> </span></div>
-    <a href='/courses/<%="#{lesson.searchable_id}"%>' class="btn button_view_card right-arrow ">VIEW LESSON </a>
+    <a href='/courses/<%="#{lesson.searchable_id}"%>' class="btn button_view_card right-arrow" onclick= "checkUser();">VIEW LESSON </a>
   </div>  
 </div> 
+<script type="text/javascript">
```

Comment:
We can move all these script files into the app/javascript/packs/ folder.  Then we just need to load that file in the views that use it.  You can see an example in app/views/courses/course_lesson_form.html.erb and app/javascript/packs/courses.js.  You'll probably have to make the file for users.js though.